crates/symbolicator/src/services/symbolication.rs

```diff
@@ -2118,6 +2118,12 @@ impl SymbolicationActor {
         sources: Arc<[SourceConfig]>,
         options: RequestOptions,
     ) -> Result<CompletedSymbolicationResponse, SymbolicationError> {
+        // check if the minidump starts with multipart form data and discard it if so
+        let minidump_path = minidump_file.to_path_buf();
+        let minidump = ByteView::open(minidump_path).unwrap_or_else(|_| ByteView::from_slice(b""));
+        if minidump.starts_with(b"--") {
+            return Err(anyhow::anyhow!("Minidump contains multipart form data").into());
+        }
```

Comment:
I don't think we want to report the error to sentry at all, IIRC the idea of this PR is to get rid of the nightly error spikes we get currently because of a misbehaving user because it results in alert noise.

Instead we would like this specific case to be handled silently *and* emit a metric for it, so we can still observe it happening.